README.md

```diff
@@ -410,6 +410,14 @@ Unset `BASH_IT_LEGACY_PASS` to have Bash-it **return to default behavior**:
 
 * `unset BASH_IT_LEGACY_PASS`
 
+### Debugging
+
+If you encounter problems with any part of Bash-it, run the following command:
+```bash
+bash-it doctor all
+```
+This will reload your bash profile and print out logs of various parts in Bash-it.
```

Comment:
We should probably mention the other parameters (`warnings`, `errors`) here as well.

Also: Can we make `all` the default? So if you call just `bash-it doctor` then it runs `bash-it doctor all` under the hood?

Reply:
done